.github/workflows/create-demo-clusters.yml

```diff
@@ -412,6 +416,20 @@ jobs:
             --from-literal="password=${ROX_ADMIN_PASSWORD}" \
             --from-literal="central_url=${CLUSTER_API_ENDPOINT}"
 
+          echo "Deploying Monitoring..."
+          helm_args=(
+            --set persistence.type="${STORAGE}"
+            --set exposure.type="${MONITORING_LOAD_BALANCER}"
+          )
+         
+          helm dependency update "${COMMON_DIR}/../charts/monitoring"
+          envsubst < "${COMMON_DIR}/../charts/monitoring/values.yaml" > "${COMMON_DIR}/../charts/monitoring/values_substituted.yaml"
+          helm upgrade -n stackrox --install --create-namespace stackrox-monitoring "${COMMON_DIR}/../charts/monitoring" --values "${COMMON_DIR}/../charts/monitoring/values_substituted.yaml" "${helm_args[@]}"
+          rm "${COMMON_DIR}/../charts/monitoring/values_substituted.yaml"
+          echo "Deployed Monitoring..."
+
+
```

Comment:
```suggestion
```

Slight formatting nit: could you make sure that there is only one empty line between the end of the script and the next job? Thanks!

Reply:
Done